perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/gatherbev_plugin.hpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef AUTOWARE__TENSORRT_BEVDET__GATHERBEV_PLUGIN_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__GATHERBEV_PLUGIN_HPP_
+
+#include <NvInfer.h>
+#include <unistd.h>
+
+#include <cmath>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace
+{
+static const char * GATHERBEV_PLUGIN_NAME{"GatherBEV"};
```

Comment:
While using `static const char*` for plugin names and versions is fine, consider if defining these as `constexpr` strings (C++11 and later) offers any benefit. `constexpr` can offer some compiler optimization opportunities.